### ISSUE TITLE
Use multiple Digital sensors

### ISSUE THREAD

**Issue by phyrexia** [mharizanov/ESP8266_Relay_Board#12]
Should allow for more sensors, and the option to select wich one like dt11, for inside, outside temp, or a ligth sensor

**Comment by stefanoco** [comment]
It seems to me that the current firmware contained in this repo supports (up to) four ds18b20 sensors, is this correct? Is this feature also in the 1-relay version?

**Comment by mharizanov** [comment]
Yes, the same feature exists. Thermostat control is over the sensor with smallest serial number
